purchase_requisition_order_address/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,3 @@
+- Open a contact for Deco Addict, set the Order Address
+- Open Purchase > Orders > Blanket Orders
+- Create a new Blanket Order for Deco Addict, the Order Address is copied from the contact
```

Comment:
Can you also test if the order address of the contract overules the order adress of the partner when creating a purchase order?

Reply:
I updated the test instructions to reflect that it is carried from the Blanket order to the Purchase order, but maybe I don't fully understand your question. How does it overrule the order address of the partner? The field is readonly at each step, so only the order address set on the contract can be used, and can only be changed in the contact itself

Reply:
The field should be writable on every step.

If somebody changes the order partner on the requisition and creates an order, the order should copy the order partner from the requisition.

Reply:
@janikvonrotz currently, the field can be changed on the blanket order and RFQ, but not on the confirmed purchase order which has a delivery. This is the correct behavior, right?

Reply:
@janikvonrotz I also made it writable here: https://github.com/Mint-System/Odoo-Apps-Purchase-Workflow/pull/2/commits/69a6bc12bd5268772b098352f5b743a61f16e861

Reply:
@laszlokr Yes, once its confirmed it is not writable.

You can check it on the production system: https://erp.trimada.ch/